simpleocr/files.py

```diff
@@ -45,45 +32,152 @@ def write(self):
         write_boxfile(self.path, self.classes, self.segments)
 
 
-class ImageFile(object):
```

Comment:
Agreed. In a ideal scenario, we should be using composition instead of inheritance. That would make it harder on the rest of the program (mostly `ocr.py`), but might be worth it, to avoid the confusion. What do you think?